locale/en/knowledge/javascript-conventions/what-is-json.md

```diff
@@ -14,11 +14,13 @@ has become the defacto standard for the web. JSON can be represented
 as either a list of values, e.g. an Array, or a hash of properties and
 values, e.g. an Object.
 
-    // a JSON array
-    ["one", "two", "three"]
+```js
```

Comment:
What about changing to 'json' instead of 'js' here? Because this is a pure 'json' formation :)

Reply:
I mean ```json instead of ```js